src/services/services.ts

```diff
@@ -1958,6 +1958,12 @@ namespace ts {
                     (char >= CharacterCodes.A && char <= CharacterCodes.Z) ||
                     (char >= CharacterCodes._0 && char <= CharacterCodes._9);
             }
+
+            function isNodeModulesFile(path: string): boolean {
+                const node_modulesFolderName = "/node_modules/";
+
+                return path.indexOf(node_modulesFolderName) !== -1;

```

Comment:
This is simply a peeve of mine, but I don't think that checking for `-1`, specifically, makes sense.  Why not rule out any negative value (i.e. `>= 0`)?  This specific API may be documented to return exactly `-1` (I haven't checked), but many others are not.